src/registry/editgrid/empty.ts

```diff
@@ -1,13 +1,53 @@
 import type {EditGridComponentSchema} from '@open-formulieren/types';
 
-import type {IsEmpty} from '@/registry/types';
+import type {GetRegistryEntry, IsEmpty} from '@/registry/types';
 import type {JSONObject} from '@/types';
 
-const isEmpty: IsEmpty<EditGridComponentSchema, JSONObject[]> = (_componentDefinition, value) => {
-  // Similar to Formio generic isEmpty https://github.com/formio/formio.js/blob/29939fc9d66f2b95527c90d3cf7729570c3d3010/src/components/_classes/component/Component.js#L3757
-  // The value of an editgrid component must be an array with 1 or more entries.
-  const hasValue = Array.isArray(value) && value.length >= 1;
-  return !hasValue;
+/**
+ * A function that validates the editgrid children components and returns whether they
+ * are considered as empty or not.
+ *
+ * Each (child) component is validated according to its `isEmpty` implemented function.
+ *
+ * @param parentComponentDefinition The parent (editgrid) component definition.
+ * @param componentValue The value of the editgrid component.
+ *
+ * @return The function returns whether the (child) component is empty (true) or not (false).
+ */
+export function isEmptyChildComponent(
```

Comment:
```suggestion
export function isEmptyChild(
```
As the function validates the editgrid child as a whole, not an individual editgrid child component.

A name like `hasEmptyChildComponent` could also work